src/main/java/redis/clients/jedis/commands/MultiKeyCommands.java

```diff
@@ -21,6 +22,11 @@
 import java.util.Set;
 
 public interface MultiKeyCommands {
+
+  Long copy(String srcKey, String dstKey, CopyParams params);
```

Comment:
```suggestion
  boolean copy(String srcKey, String dstKey, Integer db, boolean replace);
```
WDYT?

Reply:
I agree with returning `boolean`.  but `db` is not required. people need to pass null If they don't want it. anyway They all
 fine to me. : )

Reply:
If they don't want it, they can use the other method.

Reply:
```suggestion
  boolean copy(String srcKey, String dstKey, int db, boolean replace);
```

Reply:
```suggestion
  boolean copy(String srcKey, String dstKey, int db, boolean replace);
```
Sorry, my suggestion could be even better.

Reply:
```
boolean copy(String srcKey, String dstKey, int db, boolean replace);
boolean copy(String srcKey, String dstKey, boolean replace);
```
So, all we need to do is add the above two methods. Is that right?

Reply:
yes